records_mover/records/prep.py

```diff
@@ -61,19 +62,24 @@ def prep_table_for_load(self,
                                                            db_engine=db_engine,)
             if (how_to_prep == ExistingTableHandling.TRUNCATE_AND_OVERWRITE):
                 logger.info("Truncating...")
-                db_conn.execute(text(f"TRUNCATE TABLE {schema_and_table}"))
+                meta = MetaData()
+                table = Table(self.tbl.table_name, meta, schema=self.tbl.schema_name)
```

Comment:
when constructing these table objs do you have to set the `autoload_with=engine` or anything for it to correctly initialize or no? ive always done that in the past not sure what its needed for

Reply:
`autoload_with` is use to reflect the table:
https://docs.sqlalchemy.org/en/20/core/reflection.html

I don't think we actually need to do this with this statement because we're just deleting all rows from the table; reflection adds significant time overhead to running statements